api/app/models.py

```diff
@@ -121,7 +121,7 @@ class User(db.Model):
     services = db.relationship(
         'Service',
         secondary='user_to_service',
-        backref='user_to_service')
+        backref='users')
```

Comment:
was this just a mistake? seems like this would require a few things to get cleaned up

Reply:
Intentional. It comes off this commit: https://github.com/alphagov/notifications-api/commit/33166f3fb2825b360ab19569f08414dcf52a886a looks like just a different way of doing the joining table mapping.